addons/website_event/static/src/js/website_event_ticket_details.js

```diff
@@ -0,0 +1,38 @@
+odoo.define('website_event.ticket_details', function (require) {
+    var publicWidget = require('web.public.widget');
+
+    publicWidget.registry.ticketDetailsWidget = publicWidget.Widget.extend({
+        selector: '.o_wevent_js_ticket_details',
+        events: {
+            'click .o_wevent_registration_btn': '_onTicketDetailsClick',
+            'change .custom-select': '_onTicketQuantityChange'
+        },
+        start: function (){
+            this.foldedByDefault = this.$el.data('foldedByDefault') === 1;
+            return this._super.apply(this, arguments);
+        },
+        _getTotalTicketCount: function (){
+            var ticketCount = 0;
+            this.$('.custom-select').each(function (){
+                ticketCount += parseInt($(this).val());
+            });
+            return ticketCount;
+        },
+        //--------------------------------------------------------------------------
+        // Handlers
+        //--------------------------------------------------------------------------
+        _onTicketDetailsClick: function (ev){
+            ev.preventDefault();
+            if (this.foldedByDefault){
+                $(ev.currentTarget).toggleClass('btn-primary text-left pl-0');
+                $(ev.currentTarget).siblings().toggleClass('d-none');
+                this.$('.close').toggleClass('d-none');
```

Comment:
Someone told me once: "we don't do toggle anymore in js because there are issues if people do multiple clicks, we prefer to explicitely give a parameter". As there is no RPC and therefore no real latency I suppose it is ok to keep it here ...